packages/wonder-stuff-core/src/keys.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+/**
+ * Return an array of the enumerable keys of an object.
+ *
+ * @param {$ReadOnly<{[string]: mixed}>} obj The object for which the values are
+ * to be returned.
+ * @returns {Array<$Keys<O>>} An array of the enumerable keys of an object.
+ */
+export function keys<O: {[string]: mixed}>(obj: O): Array<$Keys<O>> {
```

Comment:
Good call forcing the type of `O` to be a subtype of `{[string]: mixed}`—otherwise `keys` would not behave according to its type if `O` were typed to have non-string keys. IMO it's pretty weird that Flow lets you have types like `{[number]: string}`, since that doesn't match the runtime types.